internal/service/authserv/create.go

```diff
@@ -29,5 +32,18 @@ func (s service) Create(ctx context.Context, user models.User) (*int64, error) {
 		return nil, fmt.Errorf("user creation error: %w", err)
 	}
 
+	if err = s.cache.Create(ctx, id, user); err != nil {
+		return nil, fmt.Errorf("user add cache error: %w", err)
+	}
+
 	return &id, nil
 }
+
+func validatePass(password string) error {
```

Comment:
if len(password) < 8 || len(password) > 255 {return fmt.Errorf("password must be at least 8 characters but no more 255")}